reactive/kubernetes_master.py

```diff
@@ -1813,21 +1850,21 @@ def build_kubeconfig():
 
         # make a copy in a location shared by kubernetes-worker
         # and kubernete-master
-        create_kubeconfig(kubeclientconfig_path, local_server, ca_crt_path,
+        create_kubeconfig(kubeclientconfig_path, public_server, ca_crt_path,
```

Comment:
I'm not convinced switching to `public_server` across the board is necessary or a good idea. This is going to force all local API traffic to go through external loadbalancers. What are you seeing that makes this necessary?

Reply:
Agreed.  `local_server` is using the ingress address for the `kube-api-endpoint` relation, which seems like it should be fine.

Reply:
Yep, I've looked over this again.  I was conflating loopback address vs local address, this works fine with local_server.